### ISSUE TITLE
Merging xP3 & eval-hackathon

### ISSUE THREAD

**Issue by Muennighoff** [bigscience-workshop/promptsource#838]
I would like to get all [xP3](https://huggingface.co/datasets/bigscience/xP3) prompts merged into eval-hackathon & then have eval-hackathon be merged into main once & for all, but I'm not sure if you want that? cc @VictorSanh @stephenbach

It would include adding:
- Normal english prompts for various new & existing datasets (many PRs already open - will clean them up & request reviews when ready if someone gives me rights to do so / merge if i can & tests pass)
- Long prompts https://github.com/bigscience-workshop/promptsource/pull/823/files ; Would request reviews once ready
- Human-translated & Machine-translated prompts (Always suffixed with ht or mt, roughly 20K lines; see https://github.com/Muennighoff/promptsource/pull/47/files)

Lmk if you're okay with that & I'll get started 🙂

**Comment by sworddish** [comment]
Look forward to this PR, any updates? @Muennighoff

**Comment by Muennighoff** [comment]
> Look forward to this PR, any updates? @Muennighoff

No, but you can always get the xP3 prompts by installing from the correct branch of this promptsource fork: [https://github.com/Muennighoff/promptsource](https://github.com/Muennighoff/promptsource)
